uqcsbot/api.py

```diff
@@ -208,6 +208,16 @@ def _add_channel(self, chan_dict: dict) -> Channel:
         self._channels_by_id[chan.id] = chan
         return chan
 
+    def reload_ims(self, *args):
+        for page in self._bot.api.im.list.paginate():
+            for im in page['ims']:
+                if im['is_user_deleted']:
+                    continue
+
+                im['is_private'] = True
+                im['name'] = im['id']
```

Comment:
can probably omit this name altogether (see: https://api.slack.com/types/channel at about half way down where they talk about name being useless)